pkg/utils/common/common.go

```diff
@@ -130,8 +131,21 @@ func GenOpenAPI(inst *cue.Instance) ([]byte, error) {
 	if inst.Err != nil {
 		return nil, inst.Err
 	}
+	// Due to #1773, refine cue instance to merely include `parameter` identifier
+	paramVal := inst.LookupDef(velacue.ParameterTag)
+	sysopts := []cue.Option{cue.All(), cue.DisallowCycles(true), cue.ResolveReferences(true), cue.Docs(true)}
+	paramSyntax, err := format.Node(paramVal.Syntax(sysopts...))
```

Comment:
`paraVal` should be checked whether it's nil ojbect.

Reply:
Get it! I used `paramVal.Exists()` to check it.

Reply:
I'm sorry, by looking deep into function `func (inst *Instance) LookupDef(path string) Value` again, I can confirm it won't return a nil object. So please ignore my comment and go ahead:)

Reply:
It won't be a nil object, but you remind me that we still need to see whether there is `parameter` definition inside `cue.Instance`. So calling `Value.Exists()` should be fine here.